src/components/ColorPicker/ColorPicker.stories.tsx

```diff
@@ -0,0 +1,129 @@
+            ["#e1c4be", "40"],
+            ["#f0e1de", "20"],
+        ],
+    },
+    {
+        title: "Green",
+        colors: [
+            ["#006452", "90"],
+            ["#00866e", "75"],
+            ["#00a084", "70 (Green Berlin)"],
+            ["#00c8a5", "60"],
+            ["#80dbb7", "50"],
+            ["#bee1d4", "40"],
+            ["#def0e9", "20"],
+        ],
+    },
+    {
+        title: "Yellow",
+        colors: [
+            ["#cc9000", "90"],
+            ["#e6a200", "79"],
+            ["#ffb400", "60"],
+            ["#eec779", "50"],
+            ["#e1d4be", "40"],
+            ["#f0e9de", "20"],
+        ],
+    },
+].map((palette) => ({ ...palette, colors: palette.colors.map(addAlpha) }));
+
+const Template: Story<ColorPickerProps> = (args) => {
+    const [selectedColor, setSelectedColor] = useState(args.currentColor);
+    return (
+        <ColorPicker
+            {...args}
+            currentColor={selectedColor}
+            onSelect={(color) => {
+                args.onSelect(color);
+                setSelectedColor(color);
+            }}
+        />
+    );
+};
+
+export const OnlyCustomColor = Template.bind({});
+
+export const WithBrandColors = Template.bind({});
+
+WithBrandColors.args = { palettes };
+
+export const WithinFlyout: Story<ColorPickerProps> = ({ currentColor }) => {
+    const [temporaryColor, setTemporaryColor] = useState(currentColor);
+    const [selectedColor, setSelectedColor] = useState<ColorPickerProps["currentColor"] | null>(null);
+
+    return (
+        <div className="dark:tw-text-white">
+            <div className="tw-flex tw-items-center">
+                Some text
+                <Flyout
+                    onClick={() => setSelectedColor(temporaryColor)}
+                    onClose={() => setTemporaryColor(selectedColor || currentColor)}
+                    trigger={
+                        <MenuItem
+                            title={
+                                selectedColor
+                                    ? [selectedColor.name || "", selectedColor.value].join(" ")
+                                    : "Selected Color"
+                            }
+                            decorator={
+                                !selectedColor?.value ? (
+                                    <span className="tw-text-black-70">
+                                        <IconColors size={IconSize.Size12} />
+                                    </span>
+                                ) : (
+                                    <span
+                                        className="tw-h-4 tw-w-4 tw-rounded tw-flex tw-items-center tw-justify-center"
+                                        style={{ background: selectedColor?.value }}
+                                    />
+                                )
+                            }
+                        />
+                    }
+                >
+                    <ColorPicker
```

Comment:
There's a weird glitch where the active slider element is not rendered in the right position but transitions there from some place at the bottom-left of the component.